app/javascript/components/components/Nav.js

```diff
@@ -173,7 +173,7 @@ export default function PrimarySearchAppBar(props) {
           </div>
           <div className={classes.grow} />
           <div className={classes.sectionDesktop}>
-
+{/*             
             <IconButton
               edge="end"
```

Comment:
What is going on here?

Reply:
We were trying to decide if we wanted to use the icons or not for the user profile in the nav so we commented them out while working through a merge conflict with Erik.